src/Helpers/PageHelpers/PageHelper.cs

```diff
@@ -33,24 +34,30 @@ public class PageHelper : IPageHelper
         private readonly IWebHostEnvironment _environment;
         private readonly FormConfiguration _disallowedKeys;
         private readonly IDistributedCacheWrapper _distributedCache;
+        private readonly IEnumerable<IFileStorageProvider> _fileStorageProviders;
         private readonly IEnumerable<ILookupProvider> _lookupProviders;
         private readonly DistributedCacheExpirationConfiguration _distributedCacheExpirationConfiguration;
+        private readonly IConfiguration _configuration;
 
         public PageHelper(IViewRender viewRender, IElementHelper elementHelper, IDistributedCacheWrapper distributedCache,
             IOptions<FormConfiguration> disallowedKeys, IWebHostEnvironment enviroment,
             IOptions<DistributedCacheExpirationConfiguration> distributedCacheExpirationConfiguration,
             ISessionHelper sessionHelper, IEnumerable<ILookupProvider> lookupProviders,
-            IActionHelper actionHelper)
+            IActionHelper actionHelper,
+            IEnumerable<IFileStorageProvider> fileStorages,
+            IConfiguration configuration)
```

Comment:
Can we rename the fileStorages here too? Also maybe have each parameter on a single line just for consistency